### DISCUSSION TITLE
Provide newer libvirt/qemu versions?

### DISCUSSION THREAD

**Discussion by fkr** [SovereignCloudStack/issues#82] [Ideas]
On May 11th in the Team IaaS meeting @berendt raised an issue regarding the idea to provide newer libvirt/qemu versions.
See [this issue](https://github.com/osism/issues/issues/169) in the osism repo.

What does the community (the SCS operators and SCS integrators) think about this? Would you like to see the benefits of newer versions being available via SCS/osism?

One example that @berendt pointed out is SGX, however there are more use-cases for newer versions out there and from my perspective it would be worthwhile to pursue this and see how much effort this would really mean.

What are your opinions?

**Comment by garloff** [comment]
I would avoid building our own version for common packages.
Package maintenance is real work and to do it well, you need to have policies in place how you deal with bugs, security issues, what you upgrade strategy is (compatibility?) etc. This is what Linux distributors do and have -- over the years -- learnt to do well. You need experience and it consumes real (wo)manpower.
If we can find some *well-maintained* version that meets our needs, we should use it.
Doing our own package maintenance should be the rare exception and needs good justification, IMVHO.
